### ISSUE TITLE
Launching the electron app in debug mode does not work anymore.

### ISSUE THREAD

**Issue by kittaakos** [eclipse-theia/theia#8165]
### Steps to Reproduce:

1. Build the app
2. `yarn rebuild:electron`
3. Launch the `Launch Electron Backend` config.

```
/Users/akos.kitta/git/theia/node_modules/.bin/electron . --log-level=debug --hostname=localhost --no-cluster --app-project-path=/Users/akos.kitta/git/theia/examples/electron --remote-debugging-port=9222 --no-app-auto-install --plugins=local-dir:../../plugins
internal/modules/cjs/loader.js:584
    throw err;
    ^

Error: Cannot find module '"/Applications/Visual'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at Module._preloadModules (internal/modules/cjs/loader.js:824:12)
    at preloadModules (internal/bootstrap/node.js:601:7)
    at startup (internal/bootstrap/node.js:273:9)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:622:3)
Process exited with code 1
```

Somewhere a path is not escaped, I have `/Applications/Visual Studio Code.app` but I do not know why is it needed suddenly.

### Additional Information

- Operating System:
- Theia Version: a1ea2307d

**Comment by akosyakov** [comment]
Do you run it from VS Code or Theia?

**Comment by kittaakos** [comment]
> Do you run it from VS Code or Theia?

From VS Code.

**Comment by kittaakos** [comment]
It's [this](https://github.com/microsoft/vscode/issues/102035#issuecomment-656801666). Yet another good reason to bump the Node.js version.

**Comment by kittaakos** [comment]
Closing, there is a workaround.

**Comment by connor4312** [comment]
A fix for this issue has now been released in VS Code 1.47.2. If you previously had to opt-out of the new debugger, you should be safe to turn it back on. Thanks again for the report!
